lib/servo.js

```diff
@@ -275,8 +274,8 @@ Servo.prototype.to = function(degrees, time, rate) {
   if (this.invert) {
     degrees = Board.map(
       degrees,
-      this.range[0], this.range[1],
-      this.range[1], this.range[0]
+      0, 180,
+      180, 0
```

Comment:
Sorry, saw your second note after I replied. Yes, we really have two ranges. Global coordinate range and local/device coordinate range (typically 0-180).